examples/workspace-tags/README.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Sample Template with Workspace Tags
+description: Review the sample template and introduce dynamic workspace tags to your template
+tags: [local, docker, workspace-tags]
+icon: /icon/docker.png
+---
+
+# Overview
+
+This Coder template presents use of [Workspace Tags](https://coder.com/docs/v2/latest/templates/workspace-tags) [Coder Parameters](https://coder.com/docs/v2/latest/templates/parameters).
+
+# Use case
+
+Template administrators can use static tags to control workspace provisioning, limiting it to specific provisioner groups. However, this restricts workspace users from choosing their preferred workspace nodes.
```

Comment:
Allowing users to choose a workspace node is still possible for docker and Kubernetes-based workspaces. For example, we can choose a region[node] for our dogfood workspaces. 

I am not convinced by the argument that it was not possible before to choose workspace compute nodes. Perhaps we should state that dynamic tags provide another solution (more secure as credentials live on the tagged provisioned) and optionally allow using a local provisioner when traveling.

Reply:
Rephrased